fighthealthinsurance/middleware/AuditLoggingMiddleware.py

```
@@ -0,0 +1,167 @@
+        if start_time:
+            response_time_ms = int((time.time() - start_time) * 1000)
+
+        # Extract resource info
+        resource_info = _extract_resource_info(request.path, response)
+
+        # Extract search query if present
+        search_query = None
+        if request.method == "GET":
+            search_query = request.GET.get("search") or request.GET.get("q")
+        elif request.method == "POST" and hasattr(request, "data"):
+            search_query = getattr(request.data, "get", lambda x: None)("search")
```

Ruff (0.14.8)

Unused lambda argument: `x`

(ARG005)